presto-main/src/main/java/com/facebook/presto/operator/scalar/ArrayPositionFunction.java

```diff
@@ -131,7 +132,7 @@ public static long arrayPosition(
                 Slice arrayValue = type.getSlice(array, i);
                 try {
                     Boolean result = (Boolean) equalMethodHandle.invokeExact(arrayValue, element);
-                    checkNotIndeterminate(result);
+                    checkArgument(result != null, "Array element should not be nulll");
```

Comment:
nit: This check rather looks like an assertion. No matter what arguments are passed to this function the equals function should never return `null`, as the `null` case is explicitly checked before. For such assertions usually `verify` is preferred. (same for others).

Reply:
InteliJ would give warning on `if (result)` about null pointer exception when using `verify`. It's slightly annoying to look at but that's fine I suppose.

Reply:
It is interesting why Intellij understands `checkArgument` but doesn't undestand `verify` =\